app/api/event_invoices.py

```diff
@@ -42,11 +42,10 @@ def query(self, view_kwargs):
         query_ = self.session.query(EventInvoice)
         query_ = event_query(self, query_, view_kwargs)
         if view_kwargs.get('user_id'):
-            user = safe_query(self, User, 'id', view_kwargs['user_id'], 'user_id')
+            user = safe_query(User, 'id', view_kwargs['user_id'], 'user_id')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'parameter_name' in function call](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5526478)